src/backend/replication/walsender.c

```diff
@@ -1260,6 +1261,7 @@ StartLogicalReplication(StartReplicationCmd *cmd)
 {
 	StringInfoData buf;
 	QueryCompletion qc;
+        XLogReaderRoutine xlr;
```

Comment:
<img width="486" alt="image" src="https://github.com/neondatabase/postgres/assets/8150127/5322d8a5-848d-4183-b19d-397247bcd2c8">

spaces and tabs